src/test/java/uk/gov/hmcts/reform/cwrdapi/service/impl/StaffRefDataServiceImplTest.java

```diff
@@ -0,0 +1,369 @@
+
+        assertThat(serviceSkills).isEmpty();
+    }
+
+    @Test
+    void should_return_null_list_of_service_skills() {
+        List<Skill> skills = null;
+        when(skillRepository.findAll()).thenReturn(skills);
+        StaffWorkerSkillResponse staffWorkerSkillResponse = staffRefDataServiceImpl.getServiceSkills();
+
+        assertThat(staffWorkerSkillResponse).isNotNull();
+
+        List<ServiceSkill> serviceSkills = staffWorkerSkillResponse.getServiceSkills();
+        assertThat(serviceSkills).isNotNull();
+
+        assertThat(serviceSkills).isEmpty();
+    }
+
+    @Test
+    void should_return_case_worker_profile_with_status_code_200() {
+
+        var pageRequest =
+                validateAndBuildPagination(20, 1,
+                        20, 1);
+
+        String searchString = "cwr";
+        ArrayList<CaseWorkerProfile> caseWorkerProfiles = new ArrayList<>();
+        CaseWorkerProfile caseWorkerProfile = buildCaseWorkerProfile();
+        caseWorkerProfiles.add(caseWorkerProfile);
+        Page<CaseWorkerProfile> pages = new PageImpl<>(caseWorkerProfiles);
+
+        when(caseWorkerProfileRepo.findByFirstNameOrLastName(searchString.toLowerCase(), pageRequest))
+                .thenReturn(pages);
+        ResponseEntity<List<SearchStaffUserResponse>> responseEntity =
+                staffRefDataServiceImpl.retrieveStaffUserByName(searchString, pageRequest);
+        assertEquals(200, responseEntity.getStatusCodeValue());
+
+        List<SearchStaffUserResponse> searchResponse =
+                responseEntity.getBody();
+        assertThat(responseEntity.getBody()).isNotNull();
+        validateSearchStaffUserResponses(searchResponse);
+    }
+
+    @Test
+    void should_return_case_worker_profile_with_status_code_200_when_missing_boolean_values() {
+
+        CaseWorkerProfile caseWorkerProfile = buildCaseWorkerProfile();
+
+        caseWorkerProfile.setSuspended(null);
+        caseWorkerProfile.setTaskSupervisor(null);
+        caseWorkerProfile.setCaseAllocator(null);
+        caseWorkerProfile.setUserAdmin(null);
+
+        ArrayList<CaseWorkerProfile> caseWorkerProfiles = new ArrayList<>();
+        caseWorkerProfiles.add(caseWorkerProfile);
+        Page<CaseWorkerProfile> pages = new PageImpl<>(caseWorkerProfiles);
+
+        var pageRequest =
+                validateAndBuildPagination(20, 1,
+                        20, 1);
+        String searchString = "cwr";
+        when(caseWorkerProfileRepo.findByFirstNameOrLastName(searchString.toLowerCase(), pageRequest))
+                .thenReturn(pages);
+        ResponseEntity<List<SearchStaffUserResponse>> responseEntity =
+                staffRefDataServiceImpl.retrieveStaffUserByName(searchString, pageRequest);
+        assertEquals(200, responseEntity.getStatusCodeValue());
+
+        List<SearchStaffUserResponse> searchResponse =
+                responseEntity.getBody();
+        assertThat(responseEntity.getBody()).isNotNull();
```

Comment:
1. can you add assertions for expected content in the response 
2. add assertion for total records in response header